### ISSUE TITLE
tab url

### ISSUE THREAD

**Issue by mathiashelin** [vytenizs/forkcms#17]
When you click on a tab (like in pages/edit), the url should be changed so it contains the hash. When you use this link (like pages/edit?id=1#tabTags), the right tag should automatically be shown.

**Comment by vytenizs** [comment]
Any ideas how this could be achieved?

**Comment by mathiashelin** [comment]
Yes, with JS the url can be edited an be read out to show the correct tab. I'll take a look at it.

**Comment by vytenizs** [comment]
Shouldnt, bootstrap have this functionality by default?

**Comment by mathiashelin** [comment]
I don't think it has. At least not the whole url thing. If so, can you send me a link to the documentation?

**Comment by vytenizs** [comment]
What ive found is this http://aidanlister.com/2014/03/persisting-the-tab-state-in-bootstrap/
No default functionality.

**Comment by vytenizs** [comment]
This is also a solution http://ck.kennt-wayne.de/2012/dec/twitter-bootstrap-how-to-fix-tabs
